package.json

```diff
@@ -86,12 +86,6 @@
     "@commitlint/config-conventional": "^15.0.0",
     "@rollup/plugin-commonjs": "^21.0.1",
     "@rollup/plugin-node-resolve": "^13.0.6",
-    "@semantic-release/changelog": "^6.0.1",
```

Comment:
@nschonni are you sure these are not needed? I kind of remember to have to have them of semantic-release to work but I can be wrong.

Reply:
It looks like the Action installs/loads them https://github.com/htmlhint/HTMLHint/runs/4402409850?check_suite_focus=true#step:6:31
There are just the 2 that weren't included as the base that are explicitly installed in the job